docs/README.md

```diff
@@ -53,6 +55,9 @@ Images are currently hosted in the grafana/website repo.
 
 ## Deploy changes to grafana.com
 
-When a PR is merged to main with changes in the `docs/sources` directory, those changes are automatically synced to the grafana/website repo and published to the staging site.
+When a PR is merged with changes in the `docs/sources` directory, those changes are automatically synced by a GitHub action (`.github/workflows/publish.yml`) to the grafana/website repo.
+
+* A PR targeting the `main` branch will sync to the `content/docs/grafana/next` directory in the website repo, and publish to https://grafana.com/docs/grafana/next/.
+* A PR targeting the latest/current release branch will sync to the `content/docs/grafana/latest` directory in the website repo, and publish to https://grafana.com/docs/grafana/latest/.
 
-Generally, someone from marketing will publish to production each day: so as long as the sync is successful your docs edits will be published. Alternatively, you can refer to [publishing to production](https://github.com/grafana/website#publishing-to-production-grafanacom) if you'd like to do it yourself.
+Once the sync is complete, the website will automatically publish to production - no further action is needed.
```

Comment:
```suggestion
After the sync is complete, the website will automatically publish to production.
```
What does "the website will automatically publish to production" mean? Where is that happens?